CHANGELOG.md

```diff
@@ -4,8 +4,14 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
+* Fix `Attribute` snippet (PR: [#6879](https://github.com/dotnet/vscode-csharp/pull/6879))
+* Add grammar support for new Razor directives: rendermode, preservewhitespace and typeparam (PR: [#6887](https://github.com/dotnet/vscode-csharp/pull/6887))
+* Add Razor inlay hint and inlay hint resolve handlers (PR: [#6857](https://github.com/dotnet/vscode-csharp/pull/6857))
```

Comment:
Thanks for adding this, but without a razor insertion they don't actually do anything. Was gonna mention it when we insert. Up to you if you want to leave it though, the statement is technically correct :)

Reply:
Ah I didn't realize, will remove